recipe/meta.yaml

```diff
@@ -1,45 +1,54 @@
 {% set name = "streamlit-aggrid" %}
-{% set version = "0.3.3" %}
+{% set version = "0.3.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d96edb102df98450f3b6bcf5a52b64e6ceef9678e82e2dba9cfef074e7f134ab
+  sha256: d4b776e4e7c2a91808744873843c8236065e263c22cc39e7267383ccc81486a3
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
   number: 0
+  # poetry not available for py312
+  # streamlit not available for s390x
+  skip: true  # [py<38 or py>311 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
     - poetry
-    - python >=3.7,<4.0
+    - python
   run:
+    - python
     - pandas >=1.2
-    - python >=3.7,<4.0
     - python-decouple >=3.6
```

Comment:
python-decouple is not used

Reply:
Its module is imported as `decouple` here https://github.com/search?q=repo%3APablocFonseca%2Fstreamlit-aggrid%20decouple&type=code

see usage  https://pypi.org/project/python-decouple/#usage

Also, `altair` is specified as a dep but not used (it is brought in by `streamlit` with `altair >=4.0,<6`, maybe the upstream author wants to cap the version to a specific one). In case I can add it here as well.

Reply:
good point, I think adding it as a `run_constrained` will do.

Reply:
it's not mentioned in https://inspector.pypi.io/project/streamlit-aggrid/0.3.4
so I will try without it.